docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +386,150 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+  A more illustrative but very rare example would be Tourette's with the rare swearing tic that cannot be "turned off".
+* issues of systemic exclusion or discrimination should be addressed when apparent, even at the cost of some community discomfort.
+  E.g., meetings should be moved to less convenient times, or processes made more "formal",
+  if not doing so would exclude others in a discriminatory fashion.
+
+Behaviour that is clearly racist, sexist, etc, is always a CoC violation, and never part of trade-offs.
+
+
+A.2 Guidance on applying the CoC for resource allocation decisions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section deals with resource allocation decisions by ``sktime`` community members,
+and criteria for decision making on resource allocation violating the code of conduct.
+
+Scope
+^^^^^
+
+In scope for this section are all resource allocation decisions that satisfy at least one of the following criteria:
+
+* any decisions that allocate resources primarily obtained due to the ``sktime`` project and the volunteer contributions of the community.
+  This includes research grants awarded directly for the benefit of the ``sktime`` project,
+  commercial opportunities arising from or directly directed to the ``sktime`` project,
+  and any activities that use the ``sktime`` brand as a primary branding.
+  This condition applies even if relevant decisions are made in personal roles, or roles other than ``sktime`` official roles.
+* any decisions by members of the ``sktime`` developer community, in their execution of an official ``sktime`` role.
+  This especially includes core developer, Community Council, and Code of Conduct Committee roles.
+
+Examples for in-scope:
+
+* an academic allocating grant monies from a grant, or a data science consultant providing consultancy services, with a primary ``sktime`` branding
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
+* spending decisions from ``sktime`` community administered accounts
+* decisions arising from academic collaboration or commercial requests directed to the ``sktime`` community,
+  e.g., via official email, or via digital ``sktime`` discussion tools (slack, GitHub discussion etc)
+
+Best practice
+^^^^^^^^^^^^^
+
+To ensure decision making in line with the code of conduct, decision making must be:
+
+* by the ``sktime`` community, through ``sktime`` community decision making mechanisms.
+  E.g., following decision making outlined in the ``sktime`` governance document.
+* in communication transparent to the ``sktime`` community, following communication requirements outlined in the ``sktime`` governance document.
+* pursuant of and in line with the charitable mission of ``sktime`` and NumFOCUS.
+  For-profit activities must feed back the majority of benefits obtained through it
+  to the ``sktime`` community and its charitable entities.
+
```

Comment:
Cannot wrap my head around the "majority of benefits" regulation, how should that be measured? Could ultimately be very intimidating if people fear having to transfer majority of earnings.

"feed back a significant amount of benefits proportionate to the involvement of sktime in the activity" or something like that would be less intimidating

Reply:
I removed the clause since it seems to cause controversy.
(removed on the open PR https://github.com/sktime/sktime/pull/3674)

As discussed on Friday, it is indeed unclear how to measure "majority of benefits".

Instead, this should be indirectly ensured by making this subject to sktime decision making mechanisms and transparency.